volunteercore/api/volops/opportunities.py

```diff
@@ -23,26 +36,13 @@ def get_opportunities_api():
     search = request.args.get('search')
     frequency_unit = request.args.get('frequency_unit')
     frequency_modifier = request.args.get('frequency_modifier')
-
+    
+    opportunity_weighting = scoring.FunctionWeighting(get_opportunities_score)
+    with Opportunity.query.whoosh_search(opportunity_weighting) as s:
+        print(s)
+    
     if search:
         data = Opportunity.query.whoosh_search(search, or_=True)
-        if frequency_unit and frequency_modifier:
-            data = data.filter_by(
-                frequency_unit=frequency_unit,
-                frequency_modifier=frequency_modifier)
-        elif frequency_modifier:
-            data = data.filter_by(frequency_modifier=frequency_modifier)
-        elif frequency_unit:
-            data = data.filter_by(frequency_unit=frequency_unit)
-    elif frequency_unit and frequency_modifier:
-        data = Opportunity.query.filter_by(
-            frequency_unit=frequency_unit,
-            frequency_modifier=frequency_modifier)
-    elif frequency_modifier:
-        data = Opportunity.query.filter_by(
-            frequency_modifier=frequency_modifier)
-    elif frequency_unit:
-        data = Opportunity.query.filter_by(frequency_unit=frequency_unit)
```

Comment:
From what I understand, we're no longer using frequency_modifer and frequency_unit, so I removed them.